.github/workflows/snap-store-publish-to-candidate.yml

```diff
@@ -78,21 +82,78 @@ jobs:
         run: |
           rev=$(snapcraft push $SNAP_NAME --release=$CHANNEL)
           echo "revision_${ARCHITECTURE}=${rev}" >> $GITHUB_OUTPUT
-    outputs: 
-      revision_amd64: ${{ steps.publish.outputs.revision_amd64 }}
-      revision_arm64: ${{ steps.publish.outputs.revision_arm64 }}
-      version: ${{ steps.build.outputs.version }}
+
+      - name: Upload snap artifact for testing
+        uses: actions/upload-artifact@v3
+        with:
+          name: snap
+          path: ${{ steps.build.outputs.snap }}
```

Comment:
Can't we just install the snap from the store? We just published it there?

Snap-sized binaries and GitHub actions free tier don't play nice together, so we should avoid it at all cost.

Reply:
Yep, good call. I've modified this so it installs the specific revision noted as an output from the publish phase.